src/components/data-model-table/association-selection.tsx

```diff
@@ -126,7 +126,33 @@ export const AssociationSelection = <
     meta,
   };
 
-  const { tableProps, tableQuery: queryResult } = useTable<GetQuery>({
+  const searchableKeys = getSearchableKeys(associatedRecordClass);
+
+  if (searchableKeys && searchableKeys.size > 0) {
+    tableOptions['onSearch'] = (values: any) => {
+      const result = [];
+      if (!values || !values.search || values.search.length === 0) {
+        return [];
+      }
+      for (const searchableKey of searchableKeys) {
+        result.push({
+          field: searchableKey,
+          operator: 'contains',
+          value: values.search,
+        });
+      }
+      return result;
+    };
+  }
+
+  const {
+    tableProps,
+    tableQuery: queryResult,
+    searchFormProps,
+    setSorter,
```

Comment:
This seems deprecated and need to add to make it `setSorters`, and since it's not really used, should it actually be there?

Reply:
adjusted to use setSorters